typescript/src/lib/networks/network_context.ts

```diff
@@ -721,4 +544,17 @@ export class NetworkContext {
 
         return ok(lambdaIDs)
     }
+
+    // ====================================================================================================
+    //                                       Private helper functions
+    // ====================================================================================================
+    private getSharedDirectory(relativeServiceDirpath: string, kurtosisEnclaveDataVolMountpointOnServiceContainer: string): SharedDirectory {
+       
+        const absFilepathOnThisContainer = this.enclaveDataVolMountpoint + "/" + relativeServiceDirpath;
```

Comment:
Same thing here - we should use whatever the equivalent to `path.Join` is, rather than concat'ing `/` here